src/Tizen.Applications.Common/Tizen.Applications/AppControl.cs

```diff
@@ -444,6 +445,48 @@ public ExtraDataCollection ExtraData
             }
         }
 
+        /// <summary>
+        /// Gets and sets the component ID.
```

Comment:
Please update description

Reply:
I've updated the description.

Reply:
Do you done?
you need to update ` Gets and sets the component ID.`